Felienne/hedy
# Add metrics for program runs per user type #1774

**Description**

Allow admins to see program runs based on the user type: anonymous, logged-in or a student enrolled in a class. The PR should have no effect over class metrics.

**Fix for**

#1774 

**How to test**

Run Hedy locally and:
1) log in as a teacher. Navigate to the /stats/class/$class-id page and check everything works normally (graphs are populated, no errors appear, accordions work, loading data for different period loads data accordingly)
2) log in as an admin. Navigate to the /admin/stats page. Check that there is a new accordion tab for User Type metrics. You may want to generate data (through running Hedy programs) as an anonymous user, logged in user and a class member.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
